tests/Integration/Database/QueryBuilderTest.php

```diff
@@ -127,7 +127,7 @@ public function testWhereValueSubQuery()
 
     public function testWhereValueSubQueryBuilder()
     {
-        $subQuery = DB::table('posts')->selectRaw("'Sub query value'");
+        $subQuery = DB::table('posts')->selectRaw("'Sub query value'")->limit(1);
```

Comment:
We need to limit this raw select because MySQL will produce more than one row and will throw an error.